packages/webapp/components/FooterNavBar.tsx

```diff
@@ -88,7 +88,7 @@ export default function FooterNavBar({
         spring="veryGentle"
         element="nav"
         className={classNames(
-          'grid w-full grid-flow-col items-center justify-between rounded-16 px-3',
+          'grid w-full auto-cols-fr grid-flow-col items-center justify-between rounded-16 px-3',
```

Comment:
```suggestion
          'grid w-full auto-cols-fr grid-flow-col items-center justify-between rounded-16',
```
Since using `auto-cols-fr`, we could also probably remove `px-3`, as it just makes the buttons tighter.
![image](https://github.com/dailydotdev/apps/assets/1681525/11e7484c-c9d9-48e8-b0ad-13c9d755b8cf)

Reply:
It was never properly centered before
![image](https://github.com/dailydotdev/apps/assets/1681525/12cb55a2-8897-48ea-9c44-5bbf5e8d9f29)
![image](https://github.com/dailydotdev/apps/assets/1681525/068a17a9-ee3a-4a19-b2a5-0a5399bf80a3)